specifications/api/swagger.yaml

```diff
@@ -397,7 +397,11 @@ x-transactionPendingSince: &transactionPendingSince
     block: *blockReference
 
 x-transactionDepth: &transactionDepth
-  description: Current depth of the transaction in the local chain
+  description: |
+    <span style="position: relative; left: 35px; top: -21px; vertical-align: middle; background-color: rgba(142, 142, 220, 0.05); color: rgba(50, 50, 159, 0.9); margin: 0 5px; padding: 0 5px; border: 1px solid rgba(50, 50, 159, 0.1); line-height: 20px; font-size: 13px; border-radius: 2px;">
+    <strong>if:</strong> status != pending
```

Comment:
I'd actually say: if status == in_ledger :)

Reply:
yeah, good point - I was intending to do it, but status != pending is present already in swagger.yaml and wanted to stick to convention;-)

Reply:
I will change those two then to status == in_ledger

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/1583/commits/49bc06d30aa1c79a988c2f5cb8aceb411907bc7f